exercises/practice/allergies/tests/Tests.elm

```diff
@@ -1,60 +1,285 @@
 module Tests exposing (tests)
 
-import Allergies exposing (Allergy(..), isAllergicTo, toList)
+import Allergies exposing (Allergy)
 import Expect
-import List
-import Test exposing (..)
+import Test exposing (Test, describe, skip, test)
 
 
 tests : Test
 tests =
+    -- Given a number and a substance, indicate whether Tom is allergic to that substance.
```

Comment:
Don't keep the comments in, they are only present for the people building the exercise